clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go

```diff
@@ -171,6 +171,9 @@ func (w *waitForControlledPodsRunningMeasurement) start(clients *framework.Multi
 		},
 		DeleteFunc: func(obj interface{}) {
 			deleteF := func() {
+				if tombstone, ok := obj.(cache.DeletedFinalStateUnknown); ok {
+					w.handleObject(clients, tombstone.Obj, nil)
```

Comment:
Should it handle `tombstone.Obj` and then `obj`, or is here a `return` missing?

Reply:
should return - will fix